source/computer/wsl-setup.rst

```diff
@@ -1,18 +1,154 @@
 WSL 配置指南
 ============
 
+:本节贡献者: |赵志远|\（作者）、
+             |田冬冬|\（审稿）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-01-18
+:预计花费时间: 120 分钟
+
+WSL 是什么?
```

Comment:
Should we move some of the content to the operating system section?

Reply:
That's a good idea.

Reply:
How about moving the first and third paragraphs to operating system section?

Move some of the second paragraphy to "wsl commands" section?